src/pfs-sims/hod.py

```diff
@@ -4,3 +4,153 @@
 
 
 '''
+#HOD model
+from scipy.stats import norm
+from halotools.empirical_models import HodModelFactory, OccupationComponent
+from halotools.sim_manager import UserSuppliedHaloCatalog
+from halotools.empirical_models import NFWPhaseSpace, NFWProfile
+import numpy as np
+import math
+from scipy.special import erf, erfc
+from abacusnbody.data.compaso_halo_catalog import CompaSOHaloCatalog
+import asdf
+
+#Central galaxy occupation
+class CustomCentrals(OccupationComponent):
+    def __init__(self, threshold=-23, **kwargs):
+        # Call the parent constructor
+        super(CustomCentrals, self).__init__(gal_type='centrals', threshold=threshold, upper_occupation_bound=1, **kwargs)
+
+        # Define custom parameters
```

Comment:
The HOD parameters should be an input. Not a default parameter. Also clean up any unnecessary comments.